src/main.rs

```diff
@@ -57,8 +56,8 @@ impl ggez::event::EventHandler for State {
         // There are also other kyeboard and mouse functions, see:
         // https://docs.rs/ggez/0.5.1/ggez/event/trait.EventHandler.html
         match keycode {
-            KeyCode::W => { println!("W was pressed..."); }
-            _ => (), // Do nothing
+            KeyCode::W => println!("W was pressed..."),
```

Comment:
Please, in future, abstain from removing comments for self-explanatory code in this project, because people who are unfamiliar with rust are expected to contribute to this project. 

Also, temporarily disabling auto-formatter would be nice :wink: